fmralign/alignment_methods.py

```diff
@@ -363,6 +363,9 @@ def fit(self, X, Y):
             target data'''
 
         n = len(X.T)
+        if n > 5000:
```

Comment:
I'm wondering if in that case, one should not spit a warning and return the identity. This would avoid breaking code.
WDYT ?

Reply:
Are warning really enough feedback to say things didn't work as intended at all ? For production it would be a good idea but as long as its experimental it might mislead people testing it on the reason behind unefficiency of the method

Reply:
Well the warning has to be explicit enough.
 
The problem with non-production code is that it never becomes production code.